### ISSUE TITLE
Add query link for items represented in visualization

### ISSUE THREAD

**Issue by alexdryden** [omeka-s-modules/Datavis#34]
I'm investigating methods that would allow users to surface collections of items represented throughout the data visualization. 

For example, in a "Count of items with property values" bar chart, there is a desire to be able to direct the reader to all of the items represented by each bar, e.g., to an item query results page for those items. So, for example, in this bar graph, the bar for "Dissertations" (or, its label) would be a link that would be constructed approximately as:
`/items?<query_from_the_dataviz_form>&property[0][joiner]=and&property[0][property]=<property_id_from_dataviz_form>&property[0][type]=eq&property[0][text]=Dissertations`
<img width="931" alt="Screenshot 2024-11-25 at 2 28 46 PM" src="https://github.com/user-attachments/assets/41e08fb7-5a1e-49e9-bbd7-39d113830fb9">

The most straightforward way would seem to be adding the link to the labels for the chart as an`<a>` element around the svg `<text>` element for each label. Though, adding it to the bars as well may be desirable. 

I notice that this feature is suggested by [this section of code](https://github.com/omeka-s-modules/Datavis/blob/af9e8e6386fb95cd5604f7cb2d84f4b5b5bd98ea/asset/js/diagram-render/bar_chart.js#L95-L100), but I don't see an existing method to establish a url key in the dataset. 

Is there already a way to accomplish what I am suggesting? If it doesn't yet exist, would the implementation involve building the url array and adding it to the dataset to be consumed in the linked section of code?

**Comment by jimsafley** [comment]
Yes, the solution would be to build the query URL and set it to the "url" key when creating the dataset. A sticking point here is that, if the visualization has a "Search query" set, there's no way to guarantee that the page will have the correct results. A query represented by an individual bar, column, etc. is a subset of the visualization's "Search query." This kind of "subset of a subset" isn't feasible using a URL query, at least currently.

Perhaps we could accept the visualization ID in the URL query and set the subquery that way; but, until that happens, we're not going to add the query link to the diagrams.

**Comment by jimsafley** [comment]
Feature added in [v1.4.0](https://github.com/omeka-s-modules/Datavis/releases/tag/v1.4.0)
